include/yaramod/types/regexp.h

```diff
@@ -505,24 +648,52 @@ class Regexp : public String
 	*                    ^
 	* @endcode
 	*/
-	const std::string& getSuffixModifiers() const
+	std::string getSuffixModifiers() const
 	{
-		return _suffixMods;
+		if (!_suffixMods)
+			return "";
```

Comment:
Use `std::string{}` for empty string literals.

Reply:
Fixed everywhere.